apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -66,7 +66,8 @@ defmodule Aecore.Chain.Worker do
        top_hash: genesis_block_hash,
        top_height: 0,
        total_diff: Persistence.get_total_difficulty(),
-       node_is_leader: false
+       node_is_leader: false,
+       generation_fees: %{}
```

Comment:
I think it would be better to move the data stored here to blocks (to the unnamed block_info map (the one with "refs"))